src/Symfony/Component/Console/Output/StreamOutput.php

```diff
@@ -93,7 +93,16 @@ protected function doWrite($message, $newline)
     protected function hasColorSupport()
     {
         if (DIRECTORY_SEPARATOR === '\\') {
-            return false !== getenv('ANSICON') || 'ON' === getenv('ConEmuANSI') || 'xterm' === getenv('TERM');
+            return
+                defined('PHP_WINDOWS_VERSION_MAJOR')
```

Comment:
AFAIK, ``PHP_WINDOWS_VERSION_MAJOR`` goves the version of Windows where PHP was compiled, not the version where it runs